docs/requirements.txt

```diff
@@ -0,0 +1,7 @@
+mkdocs==1.5.0
```

Comment:
same as above, we don't need to specify `mkdocs` directly as `mkdocs-material` depends on it

```suggestion
```